aquarius/events/events_monitor.py

```diff
@@ -234,100 +234,29 @@ def process_block_range(self, from_block, to_block):
-        ]
-
         # event retrieval
-        all_events = self.get_all_events(from_block, to_block)
-
-        regular_event_processors = {
-            EventTypes.EVENT_METADATA_CREATED: MetadataCreatedProcessor,
-            EventTypes.EVENT_METADATA_UPDATED: MetadataUpdatedProcessor,
-            EventTypes.EVENT_METADATA_STATE: MetadataStateProcessor,
-        }
-
-        # event handling
-        for event_name, events_to_process in all_events.items():
-            if event_name == EventTypes.EVENT_TRANSFER:
-                logger.debug(
-                    f"Starting handle_transfer_ownership for {len(events_to_process)} events"
-                )
-                self.handle_transfer_ownership(events_to_process)
-            elif event_name in regular_event_processors.keys():
-                logger.debug(
-                    f"Starting handle_regular_event_processor for {len(events_to_process)} events"
-                )
-                self.handle_regular_event_processor(
-                    event_name,
-                    regular_event_processors[event_name],
-                    processor_args,
-                    events_to_process,
-                )
-            elif event_name in [
-                EventTypes.EVENT_ORDER_STARTED,
-                EventTypes.EVENT_EXCHANGE_CREATED,
-                EventTypes.EVENT_EXCHANGE_RATE_CHANGED,
-                EventTypes.EVENT_DISPENSER_CREATED,
-            ]:
-                logger.debug(
-                    f"Starting handle_price_change for {len(events_to_process)} events"
-                )
-                self.handle_price_change(event_name, events_to_process, to_block)
-            elif event_name == EventTypes.EVENT_TOKEN_URI_UPDATE:
-                logger.debug(
-                    f"Starting handle_token_uri_update for {len(events_to_process)} events"
-                )
-                self.handle_token_uri_update(events_to_process)
-            else:
-                logger.warning(f"Unknown event {event_name}")
-                logger.warning(events_to_process)
-        self.store_last_processed_block(to_block)
+        self.get_all_events(from_block, to_block)
 
     def get_all_events(self, from_block, to_block):
         logger.debug(
             f"**************Getting all events between {from_block} to {to_block}"
         )
-        all_events = {
-            EventTypes.EVENT_TRANSFER: [],
-            # "regular" events
-            EventTypes.EVENT_METADATA_CREATED: [],
-            EventTypes.EVENT_METADATA_UPDATED: [],
-            EventTypes.EVENT_METADATA_STATE: [],
-            # price changed events
-            EventTypes.EVENT_ORDER_STARTED: [],
-            EventTypes.EVENT_EXCHANGE_CREATED: [],
-            EventTypes.EVENT_EXCHANGE_RATE_CHANGED: [],
-            EventTypes.EVENT_DISPENSER_CREATED: [],
-            #
-            EventTypes.EVENT_TOKEN_URI_UPDATE: [],
-        }
-
         if from_block >= to_block:
```

Comment:
process_block_range has this check before calling get_all_events, why repeat the check?